conda-recipe-apple/meta.yaml

```diff
@@ -29,6 +30,9 @@ requirements:
 test:
   imports:
     - slycot
+  commands:
+    - conda install nose pytest scipy parameterized --yes --quiet
```

Comment:
Do we still need `nose` and `pytest` ?

Reply:
Also, isn't there a `requires` subsection  for the `test` section (e.g., https://github.com/conda-forge/slycot-feedstock/blob/10ef9af7b10ce29ef7ba6e370ea6e0e5a1021b52/recipe/meta.yaml#L38)

Reply:
> Do we still need `nose` and `pytest` ?

no only pytest

Reply:
good idea, I was looking for that requires section....

Reply:
Yes, I like the conda recipes, as a step up to the feedstock. I use them both on osx and Linux.